.forge-snapshots/ERC6909Claims approve.snap

```diff
@@ -1 +1 @@
-46492
\ No newline at end of file
+24984
```

Comment:
please can you run tests with `forge test --isolate` not `forge test` and push the snapshots

Reply:
Done..